object/src/mem.rs

```diff
@@ -842,6 +846,20 @@ where
         }
     }
 
+    pub fn update_value_at(
```

Comment:
There is room for this method in the public API. Please don't forget to document it.